Arcanna ai 1.2.1 bugfix (#35255)
demisto/content: Packs/Arcanna/ReleaseNotes/1_2_1.md

#### Integrations
##### Arcanna.AI

> - Updated `arcanna-get-event-status` Command parameters order to match the correct and expected format

Reviewer: ```suggestion
- Fixed an issue where the `arcanna-get-event-status` command did not work as expected.
```
